profiles/kentik_snmp/raritan/raritan-pdu.yml

```diff
@@ -0,0 +1,75 @@
+#https://mibs.observium.org/mib/PDU2-MIB/
+---
+extends:
+  - system-mib.yml
+
+provider: kentik-pdu
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.6    # Raritan PX2 PDU2
+
+metrics:
+  - MIB: RARITAN-PX2-PDU2-MIB
+table:
+    OID: 1.3.6.1.4.1.13742.6.3.5.4.1
+    name: outletSensorConfigurationTable
+  symbols:
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.1
+      name: outletSensorTolerance
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.2
+      name: outletSensorMaximum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.3
+      name: outletSensorMinimum
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.4
+      name: outletSensorHysteresis
+    - OID: 1.3.6.1.4.1.13742.6.3.5.4.1.6
+      name: outletSensorUnits
```

Comment:
this looks like it would require enumeration to map the numeric value from the OID to a string value representing the units